src/tests/DataBinderTests.cpp

```diff
@@ -739,4 +751,53 @@ TEST_CASE_METHOD(SqlTestFixture, "SqlDataBinder: SqlGuid", "[SqlDataBinder],[Sql
     CHECK(!result.has_value());
 }
 
+TEST_CASE_METHOD(SqlTestFixture, "SqlDataBinder: SqlNumeric", "[SqlDataBinder],[SqlNumeric]")
+{
+    auto stmt = SqlStatement {};
+
+    UNSUPPORTED_DATABASE(stmt, SqlServerType::SQLITE); // Actually, SQLite3 does not support NUMERIC(p, s) type.
+    UNSUPPORTED_DATABASE(stmt, SqlServerType::ORACLE);
+
+    stmt.ExecuteDirect("DROP TABLE IF EXISTS Test");
+    stmt.ExecuteDirect("CREATE TABLE Test (Value NUMERIC(10, 2) NULL)");
+
+    auto const expectedValue = SqlNumeric<10, 2> { 123.45 };
+
+    INFO(expectedValue);
+    CHECK_THAT(expectedValue.ToDouble(), Catch::Matchers::WithinAbs(123.45, 0.001));
+    CHECK_THAT(expectedValue.ToFloat(), Catch::Matchers::WithinAbs(123.45F, 0.001));
+    CHECK(expectedValue.ToString() == "123.45");
+
+    stmt.Prepare("INSERT INTO Test (Value) VALUES (?)");
+    stmt.Execute(expectedValue);
+    auto const actual = stmt.ExecuteDirectSingle<SqlNumeric<10, 2>>("SELECT Value FROM Test");
+    REQUIRE(actual.has_value());
+    CHECK(*actual == expectedValue);
+
+    // Fetch and check GetColumn for GUID
+    {
+        stmt.ExecuteDirect("SELECT Value FROM Test");
+        REQUIRE(stmt.FetchRow());
+        auto const actualValue = stmt.GetColumn<SqlNumeric<10, 2>>(1);
+        CHECK(actualValue == expectedValue);
+    }
```

Comment:
In catch2 you can use [SECTION](https://github.com/catchorg/catch2/blob/devel/docs/tutorial.md#test-cases-and-sections) if you want to have also an information in the test output
```suggestion
    
    SECTION("Fetch and check GetColumn for GUID") {
        stmt.ExecuteDirect("SELECT Value FROM Test");
        REQUIRE(stmt.FetchRow());
        auto const actualValue = stmt.GetColumn<SqlNumeric<10, 2>>(1);
        CHECK(actualValue == expectedValue);
    }
```